test/testsets.json

```diff
@@ -55,7 +55,9 @@
     {
       "name": "test_crypto.js",
       "required-modules": [
-        "crypto"
+        "crypto",
+        "fs",
+        "tls"
```

Comment:
Would it make sense to create another test file, which requires the tls? because this way we will not test the crypto without tls, but we have if-defs in it.

Reply:
Great idea!